main.go

```diff
@@ -2,56 +2,52 @@ package main
 
 import (
 	"fmt"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/ldlog"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
-	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
-	"gopkg.in/launchdarkly/go-server-sdk.v5/ldcomponents"
 	"os"
 	"time"
+
+	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
```

Comment:
We now have fewer package imports: `ldcomponents` and `ldlog` were only needed because we were doing custom configuration of the client, and `ldvalue` was only needed because we had a custom user attribute.